helm/templates/deploy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: Connectivity Intake
+  namespace: bc5936-dev
+spec:
+  selector:
+    matchLabels:
+      app: connectivity-intake
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: connectivity-intake
+    spec:
+      containers:
+        - name: connectivity-intake
+          image: ghcr.io/bcgov/connectivity-intake:sha-2a147891ea52cfb6b16bd64a48234ac00d26a169
```

Comment:
you're going to want to retrieve this from your values file, e.g.: https://github.com/bcgov/cas-cif/blob/77a5d44ca1680116c033257d48ec42312247e086/chart/cas-cif/templates/app-deployment.yaml#L55

Reply:
That's a good suggestion. How would I go about referencing the most recent commit sha in the values file? I wouldn't want to hard code that in, would I?

Reply:
you wouldn't set in in your values, but in the command line that installs/upgrades your chart: 
See: https://github.com/bcgov/cas-cif/blob/9640d9c294623013d01c2a911bc9e8b2414b5cd4/Makefile#L199
And the values file: https://github.com/bcgov/cas-cif/blob/9640d9c294623013d01c2a911bc9e8b2414b5cd4/chart/cas-cif/values.yaml#L1